docs/user/aws/customization.md

```diff
@@ -1,18 +1,50 @@
 # AWS Platform Customization
 
-The following options are available when using AWS:
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, AWS-specific properties.
 
-- `machines.platform.aws.rootVolume.iops` - the reserved IOPS of the root volume
-- `machines.platform.aws.rootVolume.size` - the size (in GiB) of the root volume
-- `machines.platform.aws.rootVolume.type` - the storage type of the root volume
-- `machines.platform.aws.type` - the EC2 instance type
-- `machines.platform.aws.zones` - a list of the availability zones that the installer will use when creating machines of this pool
-- `platform.aws.region` - the AWS region that the installer will use when creating resources
-- `platform.aws.userTags` - a map of keys and values that the installer will add as tags to all resources it creates
+## Cluster-scoped properties
+
+* `amiID` (optional string): The AMI that should be used to boot machines for the cluster.
+    If set, the AMI should belong to the same region as the cluster.
+* `region` (required string): The AWS region where the cluster will be created.
+* `userTags` (optional object): Additional keys and values that the installer will add as tags to all resources that it creates.
```

Comment:
hmm, object is probably not accurate here because it is map string->string, where objects are more like `rootVolume` ie has known fields and values.

Reply:
> hmm, object is probably not accurate here because it is map string->string...

That's still [a JSON object][1] ([YAML mapping][2]), we just leave the set of valid keys more open-ended for JSON objects backed by Go `struct`s.  

[1]: https://tools.ietf.org/html/rfc8259#section-4
[2]: https://yaml.org/spec/1.2/spec.html#id2802432